common.mk

```diff
@@ -47,16 +47,14 @@ $(FIRRTL_FILE) $(ANNO_FILE): $(SCALA_SOURCES) $(sim_files)
 # create verilog files rules and variables
 #########################################################################################
 REPL_SEQ_MEM = --infer-rw --repl-seq-mem -c:$(MODEL):-o:$(TOP_SMEMS_CONF)
-HARNESS_REPL_SEQ_MEM = --infer-rw --repl-seq-mem -c:$(MODEL):-o:$(HARNESS_SMEMS_CONF)
+HARNESS_REPL_SEQ_MEM = -thconf $(HARNESS_SMEMS_CONF)
```

Comment:
I think this variable needs to change now that it's being used differently.

Reply:
I updated it, thanks